client/src/App.js

```diff
@@ -16,15 +16,17 @@ import "./App.css";
 
 function App() {
   const [state, dispatch] = useReducer(reducer, {}, initState);
-
   return (
     <userContext.Provider value={state.user}>
       <MuiThemeProvider theme={theme}>
         <BrowserRouter>
           <Switch>
             <Route path="/signup" component={SignUp} />
             <Route path="/login" component={Login} />
-            <Route path="/onboard" component={OnBoard} />
+            <Route path="/onboard" dispatch={dispatch} component={OnBoard} />
+            {/* <ProtectedRoute condition={() => state.user.id} path="/onboard">
+              <OnBoard />
+            </ProtectedRoute> */}
```

Comment:
Onboard is a protected route, user should only be able to access it after signing up. Can users access /onboard to add more languages in the future?

Reply:
I would expect this to be on /profile or a new /profile/edit page, so that other user details can be edited at the same time